### PR TITLE
first update for january

### PR DESCRIPTION
**Pull request by schimih**
Hello,
this is the first update for January in the MultiversX Ecosystem.

### CODE REVIEW

File: data/ecosystems/m/multiversx-elrond.toml [hunk 1]
```diff
@@ -1164,3 +1164,30 @@ url = "https://github.com/yum0e/my-simple-explanations"
...
+url = "https://github.com/multiversx/mx-exchange-service"
+
+[[repo]]
+url = "https://github.com/multiversx/mx-template-nextjs-dapp"
+
+[[repo]]
+url = "https://github.com/wdcurry/mxConsole"
+
+[[repo]]
+url = "https://github.com/PeerMeHQ"
```
Comment (jubos) [inline data/ecosystems/m/multiversx-elrond.toml:1181]: This warrants it's own sub ecosystem since it is a github org.  Can you spin that one out?

**Review by jubos — CHANGES_REQUESTED** [review]
Thanks for these new repos.

File: data/ecosystems/m/multiversx-elrond.toml (hunk 1 above)
Comment (schimih) [reply data/ecosystems/m/multiversx-elrond.toml]: Done. Thanks!
:raised_hands:

**Comment by jubos** [comment]
Thanks, looks good.

**Pull request merged**